app/User.php

```diff
@@ -26,4 +26,8 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+    
+    function post() {
```

Comment:
you should always declare method visibility even if public. 
```suggestion
    public function post() {
```